src/main/scala/sbtghactions/PermissionScope.scala

```diff
@@ -16,7 +16,30 @@
 
 package sbtghactions
 
-sealed trait Permissions extends Product with Serializable
+import sbtghactions.RenderFunctions.*
```

Comment:
Can we change this to `._`

Reply:
I could. But why?
If you take a look in `sbt.Defaults`, from scala 2.12.13+ this is the prefered wildcard import. And I am using it in a lot of other places too, since intellij is reading the config and does his automatically.

```scala
    scalacOptions := {
      val old = scalacOptions.value
      if (sbtPlugin.value && VersionNumber(scalaVersion.value)
            .matchesSemVer(SemanticSelector("=2.12 >=2.12.13")))
        old ++ Seq("-Wconf:cat=unused-nowarn:s", "-Xsource:3")
      else old
    }
```

Reply:
Its mainly to make sure the code is consistent, currently `.*` is not used anywhere else in the code so it sticks out as a sore thumb.

I am not against using `.*` in general however this is better tackled when adding scalafmt in this codebase, something I am planning to propose after this PR is merged.

Reply:
Hmm in that case, is it maybe okay to leave it as is, since it is used in many places in my PR, and save the effort with the goal in mind that scalafmt will ensure the consistency soon™?

Reply:
Don't worry about it then, I will add a commit to the PR just before I merge it.